entities/static/css/entity_detail.css

```diff
@@ -54,13 +54,13 @@ body {
     color: #0062cc;
 }
 
-.proile-rating {
+.profile-rating {
     font-size: 12px;
     color: #818182;
     margin-top: 5%;
 }
 
-.proile-rating span {
+.profile-rating span {
```

Comment:
Are these classes needed at all? :)

Reply:
If they are not needed, would you mind removing them?

Reply:
Please remove them by yourself, because I am in Z.M.

Reply:
Done.